freezeyt/freezer.py

```diff
@@ -335,6 +342,14 @@ def __init__(self, app, config):
             self.cancel_tasks()
             raise
 
+    def check_config(self):
+        try:
+            config_version = int(str(self.config.get("version")).split(".")[0])
```

Comment:
is Version required? When I do not write version to config, I've got `None` and it will be catch by `except`. I am not sure if the message in exception reflect this case.

Reply:
> is Version required? When I do not write version to config, I've got `None` and it will be catch by `except`. I am not sure if the message in exception reflect this case.

lines 237 and 238 reflect it.

Reply:
aa ok 👍 my bad

Reply:
> aa ok 👍 my bad

No problem David, any check is good :). Thanks for checking it out.